python/hsfs/feature_group.py

```diff
@@ -2652,6 +2650,9 @@ def __init__(
 
         self._href = href
 
+    def _get_project_name(self):
```

Comment:
maybe instead of adding new method here, we could just add it to the base fg class (then also this can be removed: https://github.com/logicalclocks/feature-store-api/pull/1069/files#diff-4ac064b1d72d926de7a5387b7d15fa9b889ac5a05ba8bcbc9974d9de334a5435R2436)